spec/custom_enum_spec.rb

```diff
@@ -0,0 +1,112 @@
+require './enum_methods/custom_enum.rb'
+
+RSpec.describe Enumerable do
+    describe "#my_each" do
+      it " It returns each element in array when given a block " do
+        array=[1,2,3,4,5]
```

Comment:
`Go back to the Advanced Building Blocks Project and write tests for any 6 of the enumerable methods you wrote there. In this case, identify several possible inputs for each of those functions and test to make sure that your implementation of them actually makes the tests pass. Be sure to try and cover some of the odd edge cases where you can.`

The project says any six so that we chose those six to submit. Could you please point us to where the above methods are mentioned as required?

Reply:
Ahhh yes, the odin project states this but Microverse requires you to test all methods implemented... Ideally though if you are doing TDD you really should test all public methods no?

It would benefit you to do so.

Here are my guidelines:

### Crucial things to make project works

- [ ] The project has a descriptive README.md


- [ ] All the methods implemented in the Enumerables project are tested